front-end/CopyPasta/app/LoginScreen.js

```diff
@@ -1,48 +1,77 @@
 import { StatusBar } from 'expo-status-bar';
 import React,{ useState } from 'react';
-import { Button, StyleSheet, Text, TextInput, View } from 'react-native';
+import { Button, StyleSheet, Text, TextInput, View, SafeAreaView, TouchableOpacity, KeyboardAvoidingView } from 'react-native';
 
 const LoginScreen = () => {
-  const handlePress = () => console.log({code})
+  const handlePress = () => console.log(code)
   const [code, setText] = useState('');
   return (
-    <View style={styles.containerLogin}>
-      <Text styles={styles.containerHeader}>
+    <SafeAreaView style={styles.containerLogin}>
+      <View style= {{flex:.25}}/>
+      <Text style= {styles.containerHeader}>
         CopyPasta
       </Text>
-      <Text styles={styles.containerSubHeader}>
+      <Text style= {styles.containerSubHeader}>
         sharing. made easy.
       </Text>
-      <TextInput
-        placeholder="CopyPasta code here!"
-        onChangeText={code => setText(code)}
-        defaultValue={code}
-      />
-      <Button 
-        title= "Login"
-        onPress= {handlePress}
-      />
+      <KeyboardAvoidingView>
+        <TextInput style={styles.textInput}
+          placeholder="CopyPasta code here!"
+          onChangeText={code => setText(code)}
+          defaultValue={LoginScreen.code}
+        />
+      </KeyboardAvoidingView>
+      <View style= {{flex:.2}}/>
+      <TouchableOpacity style= {styles.button} onPress={handlePress}>
+        <Text style={styles.buttonText}>
+          Login
+        </Text>
+      </TouchableOpacity>
+      <View style= {{flex:.2}}/>
       <StatusBar style="auto" />
-    </View>
+    </SafeAreaView>
   );
 }
 
 const styles = StyleSheet.create({
   containerLogin: {
+    backgroundColor: "#01003b",
     flex: 1,
-    backgroundColor: '#fff',
-    alignItems: "center",
+    alignSelf: "stretch",
     justifyContent:"center",
+    alignItems:"center",
   },
   containerHeader: {
+    flex: .2,
     fontSize:48,
+    color: "#fff",
   },
   containerSubHeader: {
-    fontSize:24
+    flex: .25,
+    fontSize:22,
```

Comment:
why does a subheader have smaller font than the text input?

Reply:
you want the code to be larger font and to stand out more